examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -330,40 +366,42 @@ def run_validation_edges(validation: Dataset, params, systems, epoch):
     num_epochs = int(np.ceil(args.param_updates / len(relative_transformations)))
     np.random.seed(args.seed)
 
+    batch_size = args.batch_size
     step_inds = []
     for epoch in range(num_epochs):
         inds = np.arange(len(training.data))
         np.random.shuffle(inds)
-        step_inds.append(inds)
+        batched_inds = []
+        num_steps = int(np.ceil(len(inds) / batch_size))
```

Comment:
is this actually robust against floating point errors? not sure if len(inds) / batch_size will round off to 2.0000001 or something and

Reply:
Replace with integer arithmetic rather than floating point